packaging_automation/publish_docker.py

```diff
@@ -50,7 +50,7 @@ class ScheduleType(Enum):
                              "schedule-type": ScheduleType.regular},
     DockerImageType.postgres_12: {"file-name": "postgres-12/Dockerfile", "docker-tag": "pg12",
                                   "schedule-type": ScheduleType.regular},
-    DockerImageType.postgres_13: {"file-name": "postgres-13/Dockerfile", "docker-tag": "pg13",
+    DockerImageType.postgres_14: {"file-name": "postgres-14/Dockerfile", "docker-tag": "pg14",
```

Comment:
since we didn't yet introduce "postgres-14/Dockerfile", I think we can postpone this change, but leaving the decision to you